MapboxNavigation/FeedbackSubtypeViewController.swift

```diff
@@ -160,8 +160,8 @@ class FeedbackSubtypeViewController: FeedbackViewController {
 
     private func sendReport() {
         if selectedItems.count > 0 {
-            selectedItems.forEach { item in
-                if let uuid = self.uuid {
+            if let uuid = self.uuid {
+                selectedItems.forEach { item in
```

Comment:
Nit: A little more straightforward this way:

```suggestion
                for item in selectedItems {
```

Reply:
It's written this way because it's the call to `if let uuid = self.uuid` that causes a new event to be created and enqueued for transmitting. Then these event update calls are what append the individual subtypes to the one feedback event that's waiting to be sent.

There shouldn't be any duplication since a subcategory is added only if it isn't already in the list.

Reply:
Oh, the feedback was just about the use of `forEach` instead of a standard for loop. Not sure why GitHub highlighted the `if let` also.